packages/react-native/src/theme/theme.ts

```diff
@@ -0,0 +1,93 @@
+type FontWeight =
+  | "normal"
+  | "bold"
+  | "100"
+  | "200"
+  | "300"
+  | "400"
+  | "500"
+  | "600"
+  | "700"
+  | "800"
+  | "900"
+  | undefined;
+
+const lightTheme = {
+  // Font sizes
+  fontSizes: {
+    knock0: 11,
+    knock1: 12,
+    knock2: 14,
+    knock3: 16,
+    knock4: 18,
+    knock5: 20,
+    knock6: 24,
+    knock7: 30,
+    knock8: 36,
+    knock9: 48,
```

Comment:
Nit: can we remove the `knock` prefix here please. This is already under our specific package so is superfluous